<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IdentityAssetEditor</class>
 <widget class="QWidget" name="IdentityAssetEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>932</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="labelIdenticonID">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditIdenticonID"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRandomIdenticonID">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../libres/resources/icons.qrc">
         <normaloff>:/icons/roll_dice.svg</normaloff>:/icons/roll_dice.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_9">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item row="4" column="5" rowspan="2" colspan="2">
      <widget class="QRWidget" name="widgetQR_2" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButtonSaveIdenticonRemote">
       <property name="text">
        <string>.svg / .png</string>
       </property>
       <property name="icon">
        <iconset resource="../../libres/resources/icons.qrc">
         <normaloff>:/icons/save.svg</normaloff>:/icons/save.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>QR</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButtonSaveIdenticonIris">
       <property name="text">
        <string>.png</string>
       </property>
       <property name="icon">
        <iconset resource="../../libres/resources/icons.qrc">
         <normaloff>:/icons/save.svg</normaloff>:/icons/save.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Agent</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Remote</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="IdenticonWidget" name="widgetIdenticonAgent" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Index</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="pushButtonSaveIdenticonHub">
       <property name="text">
        <string>.svg / .png</string>
       </property>
       <property name="icon">
        <iconset resource="../../libres/resources/icons.qrc">
         <normaloff>:/icons/save.svg</normaloff>:/icons/save.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="3" rowspan="2" colspan="2">
      <widget class="EyesWidget" name="widgetEyes" native="true"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonSaveIdenticonAgent">
       <property name="text">
        <string>.svg / .png</string>
       </property>
       <property name="icon">
        <iconset resource="../../libres/resources/icons.qrc">
         <normaloff>:/icons/save.svg</normaloff>:/icons/save.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="IrisWidget" name="widgetIris" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Iris</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="horizontalSliderIrisIndex">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
       <property name="pageStep">
        <number>2</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Hub</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5" colspan="2">
      <widget class="IdenticonWidget" name="widgetIdenticonHub" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Eyes</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="2">
      <widget class="IdenticonWidget" name="widgetIdenticonRemote" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IdenticonWidget</class>
   <extends>QWidget</extends>
   <header location="global">identity/IdenticonWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QRWidget</class>
   <extends>QWidget</extends>
   <header location="global">qr/QRWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EyesWidget</class>
   <extends>QWidget</extends>
   <header location="global">expression/EyesWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IrisWidget</class>
   <extends>QWidget</extends>
   <header location="global">identity/IrisWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../libres/resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
